cli/app/app.go

```diff
@@ -299,7 +299,7 @@ func ProjectScale(p project.APIProject, c *cli.Context) error {
 
 	err := p.Scale(context.Background(), c.Int("timeout"), servicesScale)
 	if err != nil {
-		return cli.NewExitError(err.Error(), 1)
+		return cli.NewExitError(err.Error(), 0)
```

Comment:
This is needed because now I actually return an error in case of *scaling with a containerName set*… This is an ugly fix to make it work like `docker-compose` but I really think `compose` should return an exit code of `1` in any error case.